src/fsum.js

```diff
@@ -0,0 +1,60 @@
+// adapted from https://github.com/python/cpython/blob/master/Modules/mathmodule.c#L1444
+export class Adder {
+  constructor() {
+    this.partials = new Float64Array(32).fill(0);
+    this.n = 0;
```

Comment:
These should probably have underscores to indicate that they are internal variables.

```suggestion
    this._partials = new Float64Array(32).fill(0);
    this._n = 0;
```